java_v1/README.md

```diff
@@ -0,0 +1,11 @@
+Java
+
+* Simple java implementation (single threaded)
+* Docker noob here - I will try to come up with a Dockerfile later.
```

Comment:
My Java is rusty... but this is the Dockerfile I used while experimenting with Java myself:

https://github.com/alexellis/xservedbyfinder/blob/master/java_v2/Dockerfile

You should be able to use it largely unmodified.